requirements.txt

```diff
@@ -12,3 +12,4 @@ pytz==2015.4
 Flask-WTF==0.12
 markdown==2.6.2
 Flask-Script==2.0.5
+workdays==1.4
```

Comment:
Seems like [you can feed it a list of holidays](https://pypi.python.org/pypi/workdays/). (but I don't know where we'd get them from.)